controllers/articles.controller.js

```diff
@@ -29,4 +29,19 @@ exports.getAllArticles = (req, res, next) => {
         res.status(200).send({ articles });
       })
       .catch(next); 
-  };
\ No newline at end of file
+  };
+
+exports.patchArticleById = (req, res, next) => {
+    const { article_id } = req.params;
+    const { inc_votes } = req.body;
+
+    if (!inc_votes || typeof inc_votes !== 'number') {
```

Comment:
You don't need to manually check this yourself, sql will throw an error if inc-votes is not a number